config/thresholds/8-Apr-2019.yml

```diff
@@ -9,6 +9,21 @@ disposable_income_lower: 100
 disposable_income_upper: 50000
 capital_lower: 1
 capital_upper: 30000
+pensioner_capital_disregard:
+  passported: 100_000
+  # disregard to use if monthly disposable income (excluding income derived from capital) is eqqual to or below these figures
```

Comment:
Very nitpicky - there should be only one `q` in `equal`

Reply:
👍